tests/at_test.py

```diff
@@ -1,7 +1,7 @@
 # coding utf-8
 import pytest
 
-from ladybug_comfort.apparent import apparent_temperature, apparent_temperature_warning_category
+from ladybug_comfort.at import apparent_temperature, apparent_temperature_warning_category
```

Comment:
I liked the longer name better in this case. Do you feel it is a commonly used module and you want to keep it short?

Reply:
I don't know the reason for this but I know that, throughout the thermal comfort community,  it's far more common to refer to comfort models and metrics by their abbreviation (eg. it's much more common to say "PMV" and "SET" rather than "Predicted Mean Vote" or "Standard Effective Temperature"). The same it true for "AT" since it's pretty rare to see it written out as "Apparent Temperature".  For this reason, I have been using abbreviations for the names of the modules.  If we change this back at some point, I think it should be called `apparent_temperature` and all of the other modules should be changed to be their long forms. But I'm really not that excited to write out `from predicted_mean_vote import predicted_mean_vote`.